docs/standard/globalization-localization/globalization-icu.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Globalization and ICU"
+ms.date: "05/21/2020"
+ms.technology: dotnet-standard
+helpviewer_keywords:
+  - "globalization [.NET Framework], about globalization"
+  - "global applications, globalization"
+  - "international applications [.NET Framework], globalization"
+  - "world-ready applications, globalization"
+  - "application development [.NET Framework], globalization"
+  - "culture, globalization"
+  - "icu, icu on windows, ms-icu"
+---
+
+# .NET globalization and ICU
+
+In the past, the .NET globalization APIs have used different underlying libraries on different platforms. On Unix, the APIs used [International Components for Unicode (ICU)](http://site.icu-project.org/home), and on Windows, they used [National Language Support (NLS)](/windows/win32/intl/national-language-support). This resulted in some behavioral differences in a handful of globalization APIs when running applications on different platforms. Behavior differences were evident in these areas:
+
+- Cultures and culture data
+- String casing
+- String sorting and searching
+- Sort keys
+- String normalization
+- Internationalized Domain Names (IDN) support
+- Time zone display name on Linux
+
+Starting with .NET 5.0, developers have more control over which underlying library is used, enabling applications to avoid differences across platforms.
+
+## ICU on Windows
+
+Windows 10 May 2019 Update and later versions include [icu.dll](/windows/win32/intl/international-components-for-unicode--icu-) as part of the OS, and .NET 5.0 and later versions uses ICU by default. When running on Windows, .NET 5.0 and later versions tries to load `icu.dll` and if it's available, uses it for the globalization implementation.  If that library can't be found or loaded, such as when running on older versions of Windows, .NET 5 and later versions falls back to the NLS-based implementation.
+
+> [!NOTE]
+> Even when using ICU, the `CurrentCulture`, `CurrentUICulture`, and `CurrentRegion` members still use Windows operating system APIs to honor user settings.
+
+### Using NLS instead of ICU
+
+Using ICU instead of NLS may result in behavioral differences with some globalization-related operations. To revert back to using NLS, a developer can opt out of the ICU implementation. Applications can enable NLS mode in any of the following ways:
+
+- In the project file:
+
+```xml
+<ItemGroup>
+  <RuntimeHostConfigurationOption Include="System.Globalization.UseNls" Value="true" />
+</ItemGroup>
+```
+
+- In the `runtimeconfig.json` file:
+
+```json
+{
+  "runtimeOptions": {
+     "configProperties": {
+       "System.Globalization.UseNls": true
+      }
+  }
+}
+```
+
+- By setting the environment variable `DOTNET_SYSTEM_GLOBALIZATION_USENLS` to the value `true` or `1` (without quotes).
```

Comment:
```suggestion
- By setting the environment variable `DOTNET_SYSTEM_GLOBALIZATION_USENLS` to the value `true` or `1`.
```

Reply:
The backticks don't appear on the rendered page, so we don't need to say "without the quotes".